.github/workflows/app-build-and-deploy.yml

```diff
@@ -129,6 +137,44 @@ jobs:
       azure-webapp-name: ${{ inputs.azure-app-base-name }}${{ inputs.azure-app-name-postfix }}
       image-name-with-tag: ${{ needs.build-and-publish-image.outputs.docker-image-name-with-tag }}
 
+  pause-between-jobs:
+    needs: deploy-primary-app-to-azure
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pause for 5 minutes
+        run: sleep 300  # Sleep for 300 seconds (5 minutes)
```

Comment:
This will slow deploys, but I guess we have to live with it for now.

Will be interesting to see if we get timeouts once we go live. If not we may be able to reduce?